docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md

```diff
@@ -133,10 +133,10 @@ In this step, you create and configure the type provider with the EDMX connectio
 <br />
 
 ```fsharp
-type edmx = EdmxFile<"Model1.edmx", ResolutionFolder = @"<path-tofolder-that-containsyour.edmx-file>>
+type edmx = EdmxFile<"Model1.edmx", ResolutionFolder = @"<path-tofolder-that-containsyour.edmx-file>">
 
 let edmConnectionString =
-getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
+  getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
```

Comment:
I followed the suggestions for the name changes. I'm not familiar enough with `use` to be able to make that decision.